.buildkite/verify-deployment-limits.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+cd "$(dirname "${BASH_SOURCE[0]}")/.."
+
+if [[ $(grep -r --include=\*.Deployment.yaml -L limits: .) ]]; then
```

Comment:
we should also do StatefulSets and DaemonSets

Reply:
Oops, good point. Will add.